README.md

```diff
@@ -152,4 +152,4 @@ Give a ⭐️ if you like this project!
 
 This project is [MIT](./LICENSE) licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`.  Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your read